spec/std/array_spec.cr

```diff
@@ -279,7 +279,15 @@ describe "Array" do
 
     it "gets with start and count" do
       [1, 2, 3][1, 3]?.should eq([2, 3])
-      [1, 2, 3][4, 0]?.should be_nil
+    end
+
+    it "gets with range without beginning" do
+      [1, 2, 3][1..nil]?.should eq([2, 3])
+      [1, 2, 3][4..nil]?.should be_nil
+    end
+
+    it "gets with range without end" do
+      [1, 2, 3][nil..1]?.should eq([1, 2])
```

Comment:
Could you also add a spec for `[nil...0]`?

Reply:
Ditto for string.

Reply:
If I am not wrong `[nil..0]` seems to be a valid case just like `[nil..1]` right? Anything I am missing?

Reply:
Is there any significance to this additional spec... Seems to be returning just any empty array [].

Reply:
Yes, it's an edge case and it should be specified whether it returns empty array or `nil`.